packages/expo-module-template/README.md

```diff
@@ -0,0 +1,111 @@
+
+# expo-module-template
+
+`expo-module-template` module
+
+## Installation
+
+*If your app is running in [Expo](https://expo.io) then everything is already set up for you, just `import { ExpoModuleTemplate } from 'expo';`*
```

Comment:
I'm not sure if this should be here since this is only true for modules that are created by us 🤔